### ISSUE TITLE
Can't fetch buildpacks via proxy

### ISSUE THREAD

**Issue by lameventanas** [cloudfoundry/cli#1894]
**Please fill out the issue checklist below and provide ALL the requested information.**

- [x] I reviewed open and closed github issues that may be related to my problem.
- [x] I tried updating to the latest version of the CF CLI to see if it fixed my problem.
- [x] I attempted to run the command with `CF_TRACE=1` to help debug the issue.
- [x] I am reporting a bug that others will be able to reproduce.
- [x] If this is an issue for the v7 beta release, I've read through the [official docs](https://docs.cloudfoundry.org/cf-cli/v7.html) and the [release notes](https://github.com/cloudfoundry/cli/releases).

**Describe the bug and the command you saw an issue with**
I'm testing a hello world app in php (the problem happens with any language).
```
bash$ cf push php-test -m 64M
Pushing app php-test to org cloudfoundry / space development as admin...
Getting app info...
Creating app with these attributes...
+ name:       php-test
  path:       /tmp/php
+ memory:     64M
  routes:
+   php-test.bosh-lite.com

Creating app php-test...
Mapping routes...
Comparing local files to remote cache...
Packaging files to upload...
Uploading files...
 174 B / 174 B [==========================================================================================================] 100.00% 1s

Waiting for API to complete processing files...

Staging app and tracing logs...
   Downloading binary_buildpack...
   Downloading java_buildpack...
   Downloading nginx_buildpack...
   Downloading r_buildpack...
   Downloading php_buildpack...
   Downloaded java_buildpack
   Downloaded r_buildpack   
   Downloading ruby_buildpack...
   Downloaded binary_buildpack
   Downloading staticfile_buildpack...
   Downloaded nginx_buildpack
   Downloading go_buildpack...
   Downloading dotnet_core_buildpack...
   Downloaded ruby_buildpack
   Downloading python_buildpack...
   Downloaded staticfile_buildpack
   Downloading nodejs_buildpack...
   Downloaded php_buildpack 
   Downloaded go_buildpack  
   Downloaded dotnet_core_buildpack
   Downloaded python_buildpack
   Downloaded nodejs_buildpack
   Cell 37055bd0-5100-4a47-8af2-c503996347c1 creating container for instance 0778b9c8-d641-4835-b39d-48c3019d5487
   Cell 37055bd0-5100-4a47-8af2-c503996347c1 successfully created container for instance 0778b9c8-d641-4835-b39d-48c3019d5487
   Downloading app package...
   Downloaded app package (174B)
   [DownloadCache]                  WARN  Unable to download https://java-buildpack.cloudfoundry.org/jvmkill/bionic/x86_64/index.yml )
   [Buildpack]                      ERROR Detect failed with exception #<RuntimeError: Jvmkill Agent error: Unable to find cached fil>
   Jvmkill Agent error: Unable to find cached file for https://java-buildpack.cloudfoundry.org/jvmkill/bionic/x86_64/index.yml
Error unmarshalling the following into a cloud controller error: 404 Not Found: Requested route ('api.bosh-lite.com') does not exist.

FAILED
bash$ printenv |grep -i proxy |sort
http_proxy=http://185.46.212.88:10074
HTTP_PROXY=http://185.46.212.88:10074
https_proxy=http://185.46.212.88:10074
HTTPS_PROXY=http://185.46.212.88:10074
no_proxy=.internal,api.bosh-lite.com,login.bosh-lite.com,.bosh-lite.com,192.168.50.6,10.244.0.1,10.244.0.2,10.244.0.3,10.244.0.4,10.244.0.5,10.244.0.6,10.244.0.7,10.244.0.8,10.244.0.9,10.244.0.10,10.244.0.11,10.244.0.5
NO_PROXY=.internal,api.bosh-lite.com,login.bosh-lite.com,.bosh-lite.com,192.168.50.6,10.244.0.1,10.244.0.2,10.244.0.3,10.244.0.4,10.244.0.5,10.244.0.6,10.244.0.7,10.244.0.8,10.244.0.9,10.244.0.10,10.244.0.11,10.244.0.5
bash$ cf --version
cf version 6.49.0+d0dfa93bb.2020-01-07
```

**What happened**
Tried to download buildpack directly, failed.

**Expected behavior**
Download buildpack via proxy.

**Provide more context**
- Linux RHEL7
- Tried cli 6.49.0 and 7.0.0-beta.29+0d0655662.2020-02-06

**Comment by JenGoldstrich** [comment]
Hey @lameventanas 

Sorry for the delay in answering this!

It looks like your API is set to api.bosh-lite.com which is owned by Pivotal.  I think your proxy might be misconfigured.

Are you still experiencing this issue?

**Comment by JenGoldstrich** [comment]
This issue has probably gone fairly stale, I am gonna close it please feel free to re-open it if you are experiencing this still @lameventanas 

Jenna

**Comment by lameventanas** [comment]
The issue is not stale.

api.bosh-lite.com resolves to 10.244.0.34, which is correct and routable.
